lib/cli.rb

```diff
@@ -748,6 +739,26 @@ def self.parse_scopes(scope_string)
     end
   end
 
+  desc "Start a webserver serving an HTML view of a system description"
+  long_desc <<-LONGDESC
+    Starts a web server which serves an HTML view for the given system description.
+  LONGDESC
+  arg "NAME"
+  command "serve" do |c|
+    c.flag [:port, :p], type: Integer, required: false, default_value: 7585,
+      desc: "Listen on port PORT", arg_name: "PORT"
+    c.flag [:ip, :i], type: String, required: false, default_value: "localhost",
+      desc: "Listen on ip IP", arg_name: "IP"
+
+    c.action do |global_options, options, args|
```

Comment:
Unused block argument - `global_options`. If it's necessary, use `_` or `_global_options` as an argument name to indicate that it won't be used.